charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           claimName: {{ template "flink-job-cluster.fullname" . }}-savepoint
   {{- end }}
   {{- if .Values.jobManager.volumes }}
-    {{ toYaml .Values.jobManager.volumes | indent 6}}
+{{ toYaml .Values.jobManager.volumes | indent 6}}
```

Comment:
You actually _want_ this indented, and you want to use `| nindent` instead of `|indent`

Reply:
The right way to do this is with nindent.. almost always.

Reply:
sounds good. updated!